pkg/controllers/dsa/controller.go

```diff
@@ -114,114 +114,16 @@ func (c *controller) NewDaemonSet(rawObj client.Object) *apps.DaemonSet {
 		for k, v := range devicePlugin.Spec.NodeSelector {
 			nodeSelector[k] = v
 		}
-		nodeSelector["kubernetes.io/arch"] = "amd64"
+		nodeSelector["kubernetes.io/arch"] = amd64
```

Comment:
I don't know whether this question is actually related to the pr or not, but I was thinking about this matter since I was doing work for dlb plugin.!

DSA, GPU, SGX has this node selector that makes pods assigned only on amd64 machines, but
DLB, QAT, FPGA doesn't.

If I understood correctly from the 1:1 meeting with Mikko, we do not consider other architecture than amd64, right? Then, wouldn't there be a necessity to make the plugins consistent by putting nodeselector for DLB, QAT, FPGA plugins?

Reply:
That would make sense to do as a separate PR. Feel free :)

Reply:
I see.! Let me make a pr later about that.!